src/olympia/stats/tests/test_views.py

```diff
@@ -234,353 +228,428 @@ def test_addon_anonymous(self):
         self._check_it(self.public_views_gen(addon_id=5, format='json'), 403)
 
 
-class TestCSVs(ESStatsTest):
```

Comment:
Similar test cases exist in `TestExports`. I removed the class here and kept test cases that weren't duplicated.